scripts/net-ebpf-ext.guid

```diff
@@ -0,0 +1,5 @@
+; Copyright (c) Microsoft Corporation
+; SPDX-License-Identifier: MIT
+;
+; Include all Tracelog events from NetEbpfExt.sys
```

Comment:
Should ebpf-all.guid also show netebpfext.sys events?  The comment in there says "Include all eBPF for Windows events".
[docs/GettingStarted.md](https://github.com/microsoft/ebpf-for-windows/pull/1247/files#diff-6fba9c4156fbc64844843f256ed9e2936a097b790a63c412095d2edd54747b4b) line 271 says the same.
So we should probably add the guid below into the ebpf-all.guid file.